src/main/java/net/neoforged/neoforge/event/EventHooks.java

```diff
@@ -515,10 +515,9 @@ public static PlayLevelSoundEvent.AtPosition onPlaySoundAtPosition(Level level,
         return event;
     }
 
-    public static int onItemExpire(ItemEntity entity, ItemStack item) {
-        if (item.isEmpty()) return -1;
```

Comment:
The empty check should probably be retained, except that it would return 0 now.

Reply:
The empty check is pointless as the item cannot be empty here---the event is fired in an isEmpty()-protected block. The only way it could be empty was if someone patched the movement code to empty it.

Reply:
Whoops, completely missed that